infra/scripts/migrate.sh

```
@@ -172,6 +212,9 @@
     "status")
         show_status
         ;;
+    "force")
+        force_version $2
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)